agent/README.md

```diff
@@ -20,3 +31,14 @@ pip install agent-protocol
 ```sh
 npm i agent-protocol
 ```
+
+## Usage
+
+### Python
+[See the Python package](./python)
+
+### JavaScript/TypeScript
```

Comment:
You can change this 😊

Reply:
I made it match the python section. :+1: